Tropos/Formatters/TemperatureComparisonFormatter.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+@objc(TRTemperatureComparisonFormatter) class TemperatureComparisonFormatter: NSObject {
+    static func localizedStrings(fromComparison comparison: TemperatureComparison, precipitation: String, date: NSDate) -> (description: String, adjective: String) {
+        let adjective = comparison.localizedAdjective
+        let timeOfDay = NSCalendar.currentCalendar().localizedTimeOfDay(forDate: date)
+        let timeOfYesterday = NSCalendar.currentCalendar().localizedTimeOfYesterday(relativeToDate: date)
+
+        let format = comparison == .Same
+            ? NSLocalizedString("SameTemperatureFormat", comment: "")
+            : NSLocalizedString("DifferentTemperatureFormat", comment: "")
+
+        return (
+            description: String(format: format, adjective, timeOfDay, timeOfYesterday, precipitation),
+            adjective: adjective
+        )
+    }
+
+    @available(*, unavailable, message="use localizedStrings(fromComparison:precipitation:date:) instead")
```

Comment:
At this stage, this method is still used from Objective-C. Marking this as unavailable here is just to prevent it being used from Swift.